docs/root/manual/getting-started/run-signer-node.md

```diff
@@ -424,10 +515,75 @@ sudo iptables -L -v
 sudo service netfilter-persistent save
 ```
 
+:::tip
+
+Here is an example command created with the example set for `release-preprod`:
+
+```bash
+sudo iptables -A INPUT -s 192.168.1.75 -p tcp --dport 3128 -j ACCEPT
+sudo iptables -L -v
+sudo service netfilter-persistent save
+```
+
+:::
+
 ## Verify the Mithril Signer deployment
 
 :::tip
 There is a delay of `2` epochs between the registration of the signer node and its ability to generate individual signatures. This delay is further explained in the [Mithril certificate chain in depth](https://mithril.network/doc/mithril/mithril-protocol/certificates) documentation.
 
 Once this delay has passed, you should be able to observe your `PoolId` listed in some of the certificates accessible on the [`Mithril Explorer`](https://mithril.network/explorer).
 :::
+
+### Verify your Signer is registered
```

Comment:
`### Verify your signer is registered`
Also in SPO onboarding guide :slightly_smiling_face:

Reply:
This is not what has been achieved in this commit ? https://github.com/input-output-hk/mithril/pull/1071/commits/969715b050339939652f242d63813dac251a8eb7